package.json

```diff
@@ -558,6 +558,12 @@
         "category": "Db2 for i",
         "icon": "$(database)"
       },
+      {
+        "command": "vscode-db2i.jobManager.continue",
```

Comment:
I think this needs an improved id. Perhaps `vscode-db2i.jobManager.focusContinue`